package.json

```diff
@@ -1,19 +1,55 @@
 {
-  "name": "bears-team-19",
-  "version": "0.0.0",
-  "private": true,
-  "scripts": {
-    "start": "node ./bin/www"
-  },
-  "dependencies": {
-    "body-parser": "~1.18.2",
-    "cookie-parser": "~1.4.3",
-    "debug": "~3.1.0",
-    "dotenv": "^5.0.1",
-    "express": "~4.16.2",
-    "jade": "~1.11.0",
-    "morgan": "~1.9.0",
-    "node-sass-chokidar": "^1.0.1",
-    "serve-favicon": "~2.4.5"
-  }
+    "name": "bears-team-19",
+    "version": "0.0.0",
+    "private": true,
+    "scripts": {
+        "start": "node ./bin/www",
+        "lint": "./node_modules/.bin/eslint --fix . || exit 0"
+    },
+    "dependencies": {
+        "body-parser": "~1.18.2",
+        "cookie-parser": "~1.4.3",
+        "debug": "~3.1.0",
+        "dotenv": "^5.0.1",
+        "express": "~4.16.2",
+        "jade": "~1.11.0",
+        "morgan": "~1.9.0",
+        "node-sass-chokidar": "^1.0.1",
+        "serve-favicon": "~2.4.5"
+    },
+    "devDependencies": {
+        "babel-eslint": "^8.2.2",
+        "eslint": "^4.18.2",
+        "eslint-config-airbnb": "^16.1.0",
+        "eslint-plugin-import": "^2.9.0",
+        "eslint-plugin-jsx-a11y": "^6.0.3",
+        "eslint-plugin-react": "^7.7.0"
+    },
+    "eslintConfig": {
+        "env": {
+            "browser": true,
+            "es6": true
+        },
+        "rules": {
+            "brace-style": [
+                "error",
+                "stroustrup"
+            ],
+            "comma-dangle": [
+                "error",
+                "never"
+            ],
+            "no-unused-vars": [
+                "warn"
+            ],
+            "no-var": [
+                "off"
+            ],
+            "one-var": [
+                "off"
```

Comment:
I like having one variable per line. It's more readable and there's the benefit on diffs.

From airbnb's style guide:
"Why? It’s easier to add new variable declarations this way, and you never have to worry about swapping out a ; for a , or introducing punctuation-only diffs. You can also step through each declaration with the debugger, instead of jumping through all of them at once."